src/structlog/contextvars.py

```diff
@@ -98,16 +98,22 @@ def clear_contextvars() -> None:
             k.set(Ellipsis)
 
 
-def bind_contextvars(**kw: Any) -> None:
-    """
+def bind_contextvars(**kw: Any) -> Mapping[str, contextvars.Token]:
```

Comment:
I think for some reason, Sphinx is getting confused trying to resolve the occurrences of `contextvar.Token` in these typehints and docstrings, causing a spurious warning (which is getting turned into a spurious error).

I noticed the [old-(pre-Sphinx-1.0)-format `intersphinx_mapping`](https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping) was being used, so I tried updating it to the up-to-date format, but that wasn't enough to appease Sphinx. 🤷

Reply:
Meant to say, I also tried several other things (e.g. adding `from contextvars import Token` and changing the typehint to just `Token`, and when that didn't work, tried spelling these references a bunch of other ways), but nothing I tried worked, and I gave up after the better part of an hour. I wonder if it's confusing Sphinx that both `structlog` and the standard library have modules named `contextvars`?

Reply:
Unfortunately Sphinx and types is hit and miss. :(

If you can't make it work, add it to https://github.com/hynek/structlog/blob/b39a720df2a8b144ecafd9d7eedaba9d0ecdf5dc/docs/conf.py#L69-L81

Reply:
Thanks! Done in the latest revision. (After rebasing on top of latest `main`, I also had to add an entry for bare `"TLLogger"`, to ignore a new warning I was getting that is unrelated to my changes.)